src/include/rear/ui.rb

```diff
@@ -441,25 +527,78 @@ def RearConfigDialog
           "<p><b>OK</b> saves the configuration and quits while <b>Cancel</b> closes the configuration dialog without saving.<p>"
         )
 
-
       # get varibales from config
       netfs_url = [Rear.netfs_url]
       netfs_keep_old_backup = Rear.netfs_keep_old_backup
-      modules_load = deep_copy(Rear.modules_load)
-      backup_prog_include = deep_copy(Rear.backup_prog_include)
+      use_dhclient = Rear.use_dhclient
+      modules_load = Rear.modules_load
+      backup_prog_include = Rear.backup_prog_include || []
+      post_recovery_script = Rear.post_recovery_script || []
+      required_progs = Rear.required_progs || []
+      copy_as_is = Rear.copy_as_is || []
+      backup_options = Rear.backup_options
       output = Rear.output
 
+      # set defaults:
+      if RearSystemCheck.Btrfs? && backup_prog_include.empty?
+        backup_prog_include = GetMountpoints()
+      end
+
+      # Fixme: Check item by item 
+      if required_progs.empty?
+        required_progs = %w(snapper chattr lsattr)
+      end
+
+      # Fixme: Check item by item
+      if copy_as_is.empty?
+        copy_as_is = %w(/usr/lib/snapper/installation-helper /etc/snapper/config-templates/default)
+      end
+
+      if backup_options.empty?
+        backup_options = "nfsvers=3,nolock"
+      end
```

Comment:
whole this paragraph have a lot of constants. I prefer to see it as constans, so it need not to be finding inside program.